tmt/steps/execute/__init__.py

```diff
@@ -33,12 +35,18 @@ class Execute(tmt.steps.Step):
     # Internal executor is the default implementation
     how = 'tmt'
 
+    REBOOT_TOKEN_LEN = 32
```

Comment:
Let's move it to the top of the file to the other constants as well?

Reply:
Sure, no problem. I felt that having it inside the class makes more sense. It is only used inside this class and hence it made sense to encapsulate it there. Looks like most constants in tmt are placed on the module level so I will move it for consistency reasons.